config/vlan.py

```diff
@@ -136,6 +136,9 @@ def add_vlan_member(db, vid, port, untagged):
     if (is_port and clicommon.is_port_router_interface(db.cfgdb, port)) or \
        (not is_port and clicommon.is_pc_router_interface(db.cfgdb, port)):
         ctx.fail("{} is a router interface!".format(port))
+        
+    if (clicommon.interface_is_untagged_member(db.cfgdb, port) and untagged):
+        ctx.fail("{} is already untagged member!".format(port))
```

Comment:
why ctx.fail() and not click.echo(); return;?
seems like it's not a syntax error, but logic one.

Reply:
i am following the procedures in this file. all logic are also failing with ctx.fail. 
for example see line 138 - this is logic. trying to put vlan on router port.